### DISCUSSION TITLE
TorchIo to train large 2D images .

### DISCUSSION THREAD

**Discussion by zirid** [TorchIO-project/torchio#1112] [General]
Is it possible to use TorchIo to train large 2D images as patches? 
Are there any examples of using the library with 2D images ?

**Comment by fepegar** [comment]
Hi, @zirid. It should be possible. Maybe @MirenLurBarquin, @jmlipman, or @jxchen01 can give you some advice if you need it.

**Reply by MirenLurBarquin** [reply]
Hi, @zirid. I have not direct experience with 2D images. Nevertheless, as far as I remember, I went through a 3D volume taking 2D patches. Did you try to use the [patch-based pipelines](https://torchio.readthedocs.io/patches/index.html) in the documentation, and try to set to 1 the last dimension in the `patch_size` tuple argument of the sampler?
